### ISSUE TITLE
[Bug]Missing Attribute Issue in UniformSequenceReplay

### ISSUE THREAD

**Issue by wangwwno1** [astooke/rlpyt#34]
Look like there is a missing attribute in this file.
self.extract_batch will always get batch_T=None and raise error.

File: rlpyt\replays\sequence\uniform.py
```
    def sample_batch(self, batch_B, batch_T=None):
        T_idxs, B_idxs = self.sample_idxs(batch_B, batch_T)  # batch_T missing?
        return self.extract_batch(T_idxs, B_idxs, batch_T)

    def sample_idxs(self, batch_B, batch_T=None):
        batch_T = self.batch_T if batch_T is None else batch_T
        t, b, f = self.t, self.off_backward + batch_T, self.off_forward
        high = self.T - b - f if self._buffer_full else t - b - f
        T_idxs = np.random.randint(low=0, high=high, size=(batch_B,))
        T_idxs[T_idxs >= t - b] += min(t, b) + f
        if self.rnn_state_interval > 0:  # Some rnn states stored; only sample those.
            T_idxs = (T_idxs // self.rnn_state_interval) * self.rnn_state_interval
        B_idxs = np.random.randint(low=0, high=self.B, size=(batch_B,))
        return T_idxs, B_idxs
```

**Comment by astooke** [comment]
thanks!  should be fixed in: 958229584378632497b8370114ffe18bbfb50f7d

**Comment by wangwwno1** [comment]
Issue Fixed
